app/assets/src/components/views/SampleViewV2/report_table.scss

```diff
@@ -19,6 +19,19 @@
   text-align: left;
 }
 
+.expandIcon {
+  font-size: $font-size-h5;
+  border-left: 5px solid transparent;
```

Comment:
Does `margin-left: 5px` not work here?

Reply:
Margin no. But actually padding-left works and removes the need fox box-shadow, since we not longer have two borders fighting for the corner.